src/django_components/component.py

```diff
@@ -322,7 +322,7 @@ def render(self, context: Context) -> str:
         # so it can be used by nested Slots.
         root_ctx_already_defined = OUTER_CONTEXT_CONTEXT_KEY in context
         if not root_ctx_already_defined:
-            context.push({ OUTER_CONTEXT_CONTEXT_KEY: context.__copy__() })
+            context.push({OUTER_CONTEXT_CONTEXT_KEY: context.__copy__()})
```

Comment:
Agree, I wasn't sure how the `copy()` func is used, because at first I thought it was a global, but there was no global `copy`. Updated in https://github.com/EmilStenstrom/django-components/pull/437/commits/691b663ed5df929d686f346914bf5feee67f6ff0!